lib/fabrik/DirectorInstance.js

```diff
@@ -76,17 +79,15 @@ class DirectorInstance extends BaseInstance {
   }
 
   deleteRestoreFile() {
-    Promise.try(() => {
-      if (_.includes(this.manager.agent.features, 'backup')) {
-        return this.platformManager.ensureTenantId()
-          .then(space_guid => this.manager.deleteRestoreFile(space_guid, this.guid))
-          .catch(err => {
-            logger.error(`Failed to delete restore file of instance '${this.guid}'`);
-            logger.error(err);
-            throw err;
-          });
-      }
-    });
+    if (_.includes(this.manager.agent.features, 'backup')) {
```

Comment:
@sauravmndl : how are we ensuring here that only for CF platform restore files are deleted ? Is it based on the tenant_id check below or based on this if condition ?

Reply:
the tenant_id below.

Reply:
ok , got it . Looks fine to me then.